src/jennifer/model/resource.cr

```diff
@@ -104,6 +109,11 @@ module Jennifer
       # :nodoc:
       def self.superclass; end
 
+      # :nodoc:
+      def self.table_prefix
+        Inflector.underscore(to_s).split('/')[0...-1].join("_") + "_" if to_s =~ /:/
```

Comment:
this might be faster.

to_s.includes?(':')

Reply:
yes, you are right - will adjust in the future PR